generator/generator.go

```diff
@@ -560,6 +560,9 @@ func (g *Generator) generateByProtogenGoGRPC(r *PluginRequest) (*pluginpb.CodeGe
 			continue
 		}
 		generatedFile := runProtogenGoGRPC(r.genplugin, f, true)
+		if generatedFile == nil {
```

Comment:
runProtogenGoGRPC [could return nil](https://github.com/mercari/grpc-federation/blob/9d1cb726183b8af1779af95d10d46d869ac22261/generator/protoc_gen_go_grpc.go#L136-L138) if a file does not contain any services.